app/src/main/java/com/duckduckgo/app/browser/BrowserChromeClient.kt

```diff
@@ -56,7 +58,7 @@ class BrowserChromeClient @Inject constructor(
         }
 
         customView = view
-        webViewClientListener?.goFullScreen(view)
```

Comment:
The commented code here demonstrates what was possible with the previous API. Since there isn't currently a way to retrieve `requestedOrientation` from the WebView we default to `SCREEN_ORIENTATION_LANDSCAPE`.